docker-images/cross-build/Dockerfile

```diff
@@ -4,8 +4,9 @@ FROM debian:bullseye-slim
 # reproducible builds for downstream forks for Ziti contributors 
 #
 
-ARG latest_golang=1.17.2
-ARG go_distribution_file=go${latest_golang}.linux-amd64.tar.gz
+ARG TARGETARCH
+ARG golang_version=1.19.5
```

Comment:
is there any way to 'just take the latest'? this is gonna be a constant battle to keep up?

Reply:
This line sets a default value in case the build arg is not supplied. The correct Go version is the one declared in go.mod. I've updated the README for this Dockerfile to automate providing the correct Go version as this build arg and verified that it works with the currently-declared Go version, 1.19.